### DISCUSSION TITLE
Throttling an observable property

### DISCUSSION THREAD

**Discussion by waylaa** [CommunityToolkit/dotnet#821] [Q&A]
Hello. Is it possible to throttle the updates on an observable property using this library? I am trying to find an equivalent of ReactiveUI's .WhenAnyValue() alongside with .Throttle()

In my case, I have a textbox that the user can query items from a data grid. How is it possible to not update on every keystroke, and for example, update every 500 milliseconds instead?

**Comment by hailstorm75** [comment]
The `TextBox` `Text` binding has a [`Delay`](https://learn.microsoft.com/en-us/dotnet/api/system.windows.data.bindingbase.delay?view=windowsdesktop-8.0) attribute.

Try doing something like this:

```xml
<TextBox Text="{Binding MyValue, Delay=500}"/>
```

**Reply by waylaa** [reply]
I can't find any Delay property. I am using AvaloniaUI with FluentAvaloniaUI for theming alongside with this library.

I get this error:
Unable to resolve suitable regular or attached property Delay on type Avalonia.Markup.Xaml:Avalonia.Markup.Xaml.MarkupExtensions.CompiledBindingExtension
